staging/src/k8s.io/client-go/kubernetes/clientset.go

```diff
@@ -35,7 +35,6 @@ import (
 	v1alpha1rbac "k8s.io/client-go/kubernetes/typed/rbac/v1alpha1"
 	v1beta1rbac "k8s.io/client-go/kubernetes/typed/rbac/v1beta1"
 	v1beta1storage "k8s.io/client-go/kubernetes/typed/storage/v1beta1"
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
This file is generated, compare https://github.com/kubernetes/kubernetes/blob/2734f8f8924dee7aa07839580b3c06e3ab3588c4/cmd/libs/go2idl/client-gen/generators/generator_for_clientset.go#L67-L67